### ISSUE TITLE
Instructions for managing autofs service appear to be incorrect

### ISSUE THREAD

**Issue by wfsaxton** [voxpupuli/puppet-autofs#19]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 4.9.3
- Ruby: ruby 2.1.9p490 (2016-03-30 revision 54437) [x86_64-linux]
- Distribution: Oracle Linux 7.2
- Module version: 2.1.0

## How to reproduce (e.g Puppet code you use)

Add the following to your manifest (per the README):

    class { 'autofs':
      service_enable => false,
      service_ensure => 'stopped'
    }

## What are you seeing

```
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Error while eva
luating a Resource Statement, Evaluation Error: Error while evaluating a Resource Statement, Class[Autofs]:
  has no parameter named 'service_enable'
  has no parameter named 'service_ensure' at /etc/puppetlabs/code/environments/dsee/dist/profile/manifests/orabase.pp:147:
3 on node build.example.com
```

## What behaviour did you expect instead

I expected it to disable the service and ensure it is not running

**Comment by dhollinger** [comment]
That is most definitely a mistake. 

I will add the code for that shortly

**Comment by dhollinger** [comment]
Looking at this some more, to rectify the issue, I will have to make some breaking changes requiring a major version release.

I will update docs to fix what you're seeing and re-add the feature referenced in the README for 3.0.0

**Comment by dhollinger** [comment]
Fixed and will be released for use in `2.1.1`

For the addition of the feature that was previously referenced in the README.md, please see issue #21 due out with release `3.0.0`

**Comment by emerald010** [comment]
What am I doing wrong here?
# autofs
autofs::mounts:
  'home':
	mountpoint: '/mnt/nov'
	remote:     '10.x.x.x:/nov'
	options:    '-rw,intr,timeo=20'

This is configured in a node yaml file. The puppet run recognize the change but it does not implement it.
